### ISSUE TITLE
Upgrade from 0.2.11 to 0.2.13 failed.

### ISSUE THREAD

**Issue by Cyborg-888** [mynodebtc/mynode#379]
REMOVED FILE - CONTAINED KEY

Initially after the install my browser was unable to connect to mynode. After a did a ping on the address and a netstat I was able to connect again.
I have tried the 'Retry Upgrade' button but the problem is still present.
I have attached the log.

**Comment by tehelsper** [comment]
Which device are you using? Raspi 4? Try re-flashing the sd card with the latest image from mynodebtc.com/download

**Comment by Cyborg-888** [comment]
Hi,
I am running a Raspberry Pi 4, 16Gb Ram, 1TB M.2 SDD.
After pinging ports on my network I eventually got the Pi to respond and was able to log in. Following advice from mynodebtc I switched the Package Upgrade TOR to Off. Then I went to the Developer section and Reinstalled Latest Version. This worked and I got everything back working.
I will close this but hope that the above description will be of help to others.
